node/src/main/scala/org/bitcoins/node/networking/peer/PeerMessageReceiver.scala

```diff
@@ -202,9 +202,11 @@ class PeerMessageReceiver(
         //we don't want to have to request them manually
         sender.sendHeadersMessage()
         Future.successful(this)
-      case _: AddrMessage =>
+      case addr: AddrMessage =>
+        node.handlePeerGossipMessage(addr)
         Future.successful(this)
-      case _: AddrV2Message =>
+      case addr: AddrV2Message =>
+        node.handlePeerGossipMessage(addr)
```

Comment:
When you make the super type for these 2, it might be better to do

```scala

case msg: GossipAddrMessage =>
	node.handlePeerGossipMessage(msg)
	
	msg match {
		case addr: AddrMessage =>
			...
		case addr: AddrV2Message =>
			...
    }

```

this way we always handle the gossip message and then the compile will yell at us here to handle it correctly